app/jobs/sellers_helper.rb

```diff
@@ -11,8 +11,22 @@ def self.generate_seller_json(seller:)
     seller
   end
 
+
+  def self.retrieve_amount_in_card(gift_card_id:)
```

Comment:
This method is already defined if you `git checkout master` `git pull` `git checkout ryannamgung` `git rebase mater`

Reply:
In my opinion, you should create a separate GiftCardHelper and add the method there